Server/Server.cpp

```diff
@@ -106,6 +115,22 @@ void Server::_holdADatabaseValueRequest(CommunicatorMessage databaseRequest)
         apiAnswer.at(0)[requestContent.at(1)], databaseRequest.message.clientInfo);
 }
 
+void Server::_holdAScoreboardRequest(CommunicatorMessage databaseRequest)
+{
+    std::string key = _communicatorInstance->utilitaryReceiveScoreboardAsking(databaseRequest);
+    std::map<std::string, int> scoreboard;
+    std::size_t count = 0;
+    auto apiAnswer = _databaseApi.selectUsers("Banned = 0 ORDER BY " + key + " DESC");
```

Comment:
You know what type selectUsers() returns.
And make an auto on an auto and it's incomprehensible

Reply:
updated with the type